icu4c/source/i18n/unicode/measure.h

```diff
@@ -89,6 +89,14 @@ class U_I18N_API Measure: public UObject {
      */
     bool operator==(const UObject& other) const;
 
+    /**
+     * Inequality operator.  Returns true if this object is not equal to the other object.
+     * @param other the object to compare with
+     * @return true if the objects are not equal
+     * @draft ICU 74
+     */
+    inline bool operator!=(const UObject& other) const { return !operator==(other); }
```

Comment:
I would tend to write that `!(*this == other)`, but I see that this is consistent with the rest of this file, so LGTM.

Reply:
It's more explicit this way. We are not consistent in ICU, but we have used this in a number of classes.